spicy/toolchain/include/ast/types/unit-items/switch.h

```diff
@@ -60,10 +60,12 @@ inline Node to_node(Case c) { return Node(std::move(c)); }
 class Switch : public hilti::NodeBase, public spicy::trait::isUnitItem {
 public:
     Switch(std::optional<Expression> expr, const std::vector<switch_::Case>& cases, Engine e,
-           std::optional<Expression> cond, std::vector<Hook> hooks, Meta m = Meta())
-        : NodeBase(nodes(std::move(expr), std::move(cond), cases, std::move(hooks)), std::move(m)),
+           std::optional<Expression> cond, std::vector<Hook> hooks, std::optional<AttributeSet> attributes,
```

Comment:
I'm always torn if it should be `optional<AttributeSet>` or `AttributeSet`, with the latter just remaining empty if there are no attributes. We are currently inconsistent across AST nodes on this. Advantage of the latter is that access gets easier: one doesn't need to check first if there's actually a set available. Advantage of the former is that it avoids creating lots of empty `AttributeSet` instances (less of a problem in this case because there aren't many switches, but more for other node types of which we create plenty). 

It's ok to leave as is, just wanted to mention.